sleap_roots_training/__init__.py

```
@@ -0,0 +1,3 @@
+from sleap_roots_training.evaluate import *
```

Ruff (0.8.2)

`from sleap_roots_training.evaluate import *` used; unable to detect undefined names

(F403)